.github/workflows/send-slack-message.yml

```diff
@@ -0,0 +1,53 @@
+name: Send Slack Message
```

Comment:
Could you choose a more descriptive title?

Reply:
How about E.g. `Send Post Deployment Slack Message to #main-deployment channel` ?

Reply:
Just `Send Post Deployment Slack Message` ? The channel is parameterized via the webhook, right?